cabby/geo/map_processing/edge.py

```diff
@@ -0,0 +1,70 @@
+# coding=utf-8
+# Copyright 2020 Google LLC
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+'''Basic classes and functions for Wikigeo items.'''
+
+from geopandas import GeoDataFrame
+import re
+from shapely.geometry.point import Point
+from typing import Text, Dict
+
+import attr
+
+OSM_CRS = 32633  # UTM Zones (North).
```

Comment:
Is this variable used in other files? If so, please comment where & what it's used for.